tests/test_vlan.py

```diff
@@ -261,8 +260,9 @@ def test_AddMaxVlan(self, dvs, testlog):
         # create max vlan
         vlan = min_vid
         while vlan <= max_vid:
-            self.create_vlan(str(vlan))
+            self.create_vlan(str(vlan), 0)
             vlan += 1
+        time.sleep(240)
```

Comment:
240 seconds seem to be quite long time; is a smaller number causing time to fail?

Reply:
I check the test result:

```
        time.sleep(240)
    
        # check asic database
        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN")
        vlan_entries = [k for k in tbl.getKeys() if k != dvs.asicdb.default_vlan_id]
>       assert len(vlan_entries) == (4094-1)
E       AssertionError: assert 3105 == (4094 - 1)

test_vlan.py:270: AssertionError
```

It seems 240 seconds does not enough... it need more time